.all-contributorsrc

```diff
@@ -1740,6 +1740,7 @@
       "name": "Benedikt Heidrich",
       "profile": "https://github.com/benheid",
       "contributions": [
+        "bug",
```

Comment:
you should have more here, now that I see.

design, ideas -> pipelines etc
doc -> writing or fixing docstrings
example -> in the docstrings and tutorial notebooks
mentoring -> obviously, what it says
question -> on discord
review -> many PRs
tutorial -> the pydatas

Reply:
maybe separate PR to also have the reason in the commit message?

Reply:
Yeah, I forgot it. I planned to do this successively, but I will create a new PR where I add the missing ones.